package.json

```
@@ -13,7 +14,7 @@
     "jquery": "3.7.1",
     "jquery-ui": "1.14.0",
     "jquery-ujs": "1.2.3",
-    "sass": "^1.78.0"
+    "sass": "^1.79.3"
   },
   "scripts": {
```

Can we add a pre-install script to automatically run `corepack enable`? That way we don't have to do it manually anywhere
```suggestion
  "scripts": {
    "preinstall": "corepack enable",
```

---

https://docs.npmjs.com/cli/v10/using-npm/scripts#pre--post-scripts

---

I like the idea, but won't you still need to have corepack enabled/yarn installed to run this?

---

Not really. If corepack is enabled, it will catch any yarn command and try to install it (based on the version from packageManager in package.json). Before, we needed to have yarn already installed to run any script, now we need to have corepack enabled (which is already included in nodejs releases).